modules/books.js

```diff
@@ -0,0 +1,22 @@
+export default class Books {
+  constructor() {
+    this.listOfBooks = JSON.parse(localStorage.getItem('books')) || [];
+  }
+
+  addBook(title, author) {
+    // Add new book into the existing array of books
+    this.listOfBooks.push({
+      title,
+      author,
+    });
+    // Replace old object of books with the new updated one
+    localStorage.setItem('books', JSON.stringify(this.listOfBooks));
+  }
+
+  removeBook(id) {
+    // Remove the item that has index of id
+    this.listOfBooks.splice(id, 1);
+    // Replace old object of books with the new updated one
+    localStorage.setItem('books', JSON.stringify(this.listOfBooks));
+  }
```

Comment:
- [ ] your projects looks awesome, clean, and well documented 👏 , but as part of the requirements we should stick with ES6, kindly update your functions inside the book class to arrow functions instead of normal ones, for example :

```JS
//Normal function
addBook(title, author) { ... }

//ES6 function
addBook = (title, author) => { ... }
```